scripts/prepare_sector_network.py

```diff
@@ -1256,7 +1256,7 @@ def annuity_factor(v):
     return costs
 
 
-def add_generation(n, costs, existing_capacities=0, existing_efficiencies=None):
+def add_generation(n, costs, ext_carriers, existing_capacities=0, existing_efficiencies=None):
```

Comment:
If you rename it here, you also need to rename down below

Reply:
Ups, my bad! You only added `ext_carriers` here. I thought you renamed `existing_capacities` to `ext_carriers`.

BUT: `existing_capacities` is now: `Optional[dict]`. 0 is not an allowed value. Shuold be `None` instead.